src/backend/distributed/planner/deparse_shard_query.c

```diff
@@ -57,18 +58,39 @@ RebuildQueryStrings(Job *workerJob)
 {
```

Comment:
I think the function comment of `RebuildQueryStrings` is not reflecting what it does anymore (probably not in the last couple of years :))